### ISSUE TITLE
Error in build_annotations() for mm10_cpgs

### ISSUE THREAD

**Issue by elderelizabeth** [rcavalcante/annotatr#56]
Hey there!

I am encountering this error when trying to build annotations for mm10_cpgs. No such error occurs when building other mm10 annotations, and mm10_cpgs worked fine just a few weeks ago. I tried troubleshooting myself but I am at a loss and would really appreciate the help.

Thanks for this great package by the way.

> cpgs_annots = build_annotations(genome = "mm10", annotations = "mm10_cpgs")
Building CpG islands...
Error in vroom_(file, delim = delim %||% col_types$delim, col_names = col_names,  :                                    
  bad value

**Comment by jamesboot** [comment]
I get the same error using the following code:

```
annots = c('hg38_cpgs', 'hg38_basicgenes')
annotations = build_annotations(genome = 'hg38', annotations = annots)
```

Any help greatly appreciated!

**Comment by elderelizabeth** [comment]
Hey! The only way I found to "fix" this error is to re-install (I use the BiocManager package to install and manage packages) and re-load the annotatr package. However, that only seems to temporarily fix it because it eventually bugs again so I have to keep re-installing and reloading the package whenever the error occurs. Not the best fix but at least I got it working. Hopefully this helps.

BiocManager::install("annotatr", update = TRUE, ask = FALSE, force = TRUE)

library(annotatr)
